webapp/src/widgets/buttons/button.scss

```diff
@@ -1,4 +1,6 @@
 .Button {
+    --danger-button-bg-rgb: 247, 67, 67;
```

Comment:
Previously this was added to shared variables, but it did not work well in mattermost with focalboard as a plugin.